test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -450,6 +450,9 @@
             jruby-scripting-container (:scripting-container jruby-instance)
             jruby-env (.runScriptlet jruby-scripting-container "ENV")]
         (try
+         ;; Note that if this test is run in an environment where the
+         ;; variables HTTP_PROXY, HTTPS_PROXY, NO_PROXY, or their lowercase
+         ;; variants are set, this will fail.
           (is (= #{"HOME" "PATH" "GEM_HOME" "GEM_PATH"
```

Comment:
Would it make sense to just code this assertion so it would work either way, e.g., by filtering out any of the http/no proxy keys from `jruby-env` that may be present first before doing the equality comparison for the other keys we had expected to be passed through?